flyteadmin/pkg/manager/impl/named_entity_manager.go

```diff
@@ -75,12 +68,8 @@ func (m *NamedEntityManager) GetNamedEntity(ctx context.Context, request *admin.
 	return util.GetNamedEntity(ctx, m.db, request.ResourceType, request.Id)
 }
 
-func (m *NamedEntityManager) getQueryFilters(referenceEntity core.ResourceType, requestFilters string) ([]common.InlineFilter, error) {
+func (m *NamedEntityManager) getQueryFilters(requestFilters string) ([]common.InlineFilter, error) {
 	filters := make([]common.InlineFilter, 0)
-	if referenceEntity == core.ResourceType_WORKFLOW {
```

Comment:
nice, love cleaning up this code (it's also very confusing we always force a filter on workflow state even if you filter on state down below

Reply:
Yes that's confusing!